wrappers/swift/PrivacyGuardInitConfig.swift

```diff
@@ -0,0 +1,23 @@
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+//
+
+public class PrivacyGuardInitConfig {
+    private var odwPrivacyGuardInitConfig: ODWPrivacyGuardInitConfig
+
+    /**
+    Constructor which inits with the `ODWPrivacyGuardInitConfig` object.
+
+    - Parameters:
+        - privacyGuardInitConfig: `ODWPrivacyGuardInitConfig` object which would be wrapped around.
+    */
+    public init(privacyGuardInitConfig: ODWPrivacyGuardInitConfig) {
+        self.odwPrivacyGuardInitConfig = privacyGuardInitConfig
+    }
+
+    /// Returns the `ODWPrivacyGuardInitConfig` representing the corresponding ObjC object.
+    public func getODWPrivacyGuardInitConfig() -> ODWPrivacyGuardInitConfig {
```

Comment:
If we make the underlying variable a public constant, we can remove this function. That's the Swift way of implementing accessors.